src/mobile/apps/artwork/components/related_artworks/queries/auction_query.coffee

```diff
@@ -6,7 +6,8 @@ module.exports = """
       name
       href
       is_open
-      artworks(all: true, size: 50, exclude: [$id]) {
```

Comment:
Interestingly, even though these pass `all: true` _and_ `size: 50`, it would fetch _all_ works (ignoring the size param).

Reply:
Actually the combination of those two serve to fetch all (as you point out), but when those paginated requests are fanned out `size` is used (and also determines the number of pages).

So this would fetch all, with page sizes of 50.